server/service/hosts.go

```diff
@@ -893,3 +893,44 @@ func hostsReportEndpoint(ctx context.Context, request interface{}, svc fleet.Ser
 	}
 	return hostsReportResponse{Hosts: hosts}, nil
 }
+
+type osVersionsRequest struct {
+	TeamID   *uint   `query:"team_id,optional"`
+	Platform *string `query:"platform,optional"`
+}
+
+type osVersionsResponse struct {
+	CountsUpdatedAt time.Time         `json:"counts_updated_at,omitempty"`
```

Comment:
Note that `omitempty` doesn't work for `time.Time`, you have to use a pointer-to-time to have it truly omitted.

Reply:
Good catch, I always forget about this.